src/constants/common.js

```diff
@@ -1,34 +1,6 @@
-const { elevateLog, correlationId } = require('elevate-logger')
-const logger = elevateLog.init()
-
 /**
  * name : common.js
  * author : Vishnudas
  * created-date : 19-may-2022
```

Comment:
Remove this file if there is nothing in it.

Reply:
@joffinjoy sure